Where_Is_My_Wife/Assets/_Main/Scripts/Animation/PlayerAnimationController.cs

```diff
@@ -35,11 +34,26 @@ private void Awake()
 
         private void Start()
         {
-            _movementStateEvents.JumpStart.AsUnitObservable().Subscribe(Jump);
-            _movementStateEvents.Run.AsUnitObservable().Subscribe(Run);
+            _playerStateIndicator = PlayerManager.Instance.PlayerStateIndicator;
+            
+            _movementStateEvents = PlayerManager.Instance.MovementStateEvents;
+            _wallHangStateEvents = PlayerManager.Instance.WallHangStateEvents;
+            _dashStateEvents = PlayerManager.Instance.DashStateEvents;
+            
+            _movementStateEvents.JumpStart += _ => Jump();
```

Comment:
_movementStateEvents.JumpStart += Jump;

If you use it like this the unsubscription wont work cause is creating a new method when you create a lambda.

Reply:
ad0dc164f0495a7d94fe9bf4bd7a5ff7cb9a7d48